roles/analysis/tasks/analysis-c2r.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Convert2RHEL pre-analysis report
+  ansible.builtin.shell: >
+    export PATH={{ os_path }}
+    set -o pipefail;
+    convert2rhel analyze
+    {{ convert2rhel_preconv_opts }}
+    {{ convert2rhel_enable_repos_args }}
+    2>&1 | tee -a {{ log_file }}
```

Comment:
What should be value of `log_file` ? leapp has it in common role, I will it to common role, but should it be named same, what is ripu - is it just red hat in place upgrade?